src/braket/aws/aws_device.py

```diff
@@ -623,7 +622,7 @@ def get_devices(
                 f"order_by '{order_by}' must be in {AwsDevice._GET_DEVICES_ORDER_BY_KEYS}"
             )
         types = frozenset(types or AwsDeviceType)
-        aws_session = aws_session if aws_session else AwsSession()
+        aws_session = aws_session or AwsSession()
```

Comment:
Can we test and make sure `aws_session` is never a falsey value here.

Reply:
sorry i don't understand. If aws_session is False, it returns AwsSession()? this is the same behavior as before

Reply:
This behavior is very similar but can have subtle differences in what each consider "falsey values". For example, `or` handles custom objects slightly different than `if-else`(`or` includes `__bool__()` but does not include `__len__()` as a falsey value). 

I don't see that being an issue here though so `aws_session or AwsSession()` is probably fine.